packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/ImageNode.tsx

```diff
@@ -14,34 +14,42 @@
 import { ServerContext, ServerContextValue } from '@eclipse-sirius/sirius-components-core';
 import { Theme, useTheme } from '@material-ui/core/styles';
 import { memo, useContext } from 'react';
-import { Handle, NodeProps, NodeResizer, Position } from 'reactflow';
+import { Handle, NodeProps, NodeResizer, NodeResizerProps, Position } from 'reactflow';
 import { ImageNodeData } from './ImageNode.types';
 import { NodePalette } from './palette/NodePalette';
 
 const imageNodeStyle = (
-  style: React.CSSProperties,
   theme: Theme,
+  style: React.CSSProperties,
   selected: boolean,
   faded: boolean
 ): React.CSSProperties => {
   const imageNodeStyle: React.CSSProperties = { width: '100%', height: '100%', opacity: faded ? '0.4' : '', ...style };
-
   if (selected) {
-    imageNodeStyle.outline = `${theme.palette.primary.main} solid 1px`;
+    imageNodeStyle.outline = `${theme.palette.primary.light} solid 1px`;
```

Comment:
`main` or `light`?
Sometimes you set the outline of nodes to `main` and sometimes to `light`.
This time the node outline when selected is _the same_ as the NodeResize color